setup.py

```diff
@@ -20,4 +20,10 @@
         'Operating System :: OS Independent',
     ],
     long_description=open('README.rst').read(),
+    install_requires=[
+        'pyyaml>=3,<4',
+        'boto>=2.39,<4',
```

Comment:
any reason why we're capping this one at version 4 rather than 3? 

:shipit: either way